src/providers.ts

```diff
@@ -34,7 +34,7 @@ const LD_MODE: DocumentFilter = {
 export function register(ctx: ExtensionContext, config: Configuration, flagStore: FlagStore, api: LaunchDarklyAPI) {
 	const flagView = new LaunchDarklyTreeViewProvider(api, config, flagStore, ctx);
 	if (getIsTreeviewEnabled()) {
```

Comment:
Can we use the `config` object passed to the provider here and in other locations that need to know if the tree view is enabled? We originally removed these helpers methods to reduce the amount we rely on global state. I wonder if we can use the existing `onDidChangeConfiguration` listener in `extension.ts` to handle disabling/enabling the tree view